src/scss/_search.scss

```diff
@@ -8,8 +8,9 @@
     border-radius: 10px;
   
     input {
-      padding: 10px;
+      padding: 15px;
       border: none;
+      width: 85%;
```

Comment:
This is really close! I would actually recommend a fix closer to: 
```
width: 100%;
box-sizing: border-box;
```
The width will make sure the input fills the entire parent container, and the box-sizing attribute will make sure it does not overflow.

Let me know if you have questions, and thanks for contributing